### PR TITLE
[Fixes #29] Updated README with successful post request

### LINKED ISSUE #29: Add output of successful post request (in readme)
In readme.md add output of successful post request.
Also adding some other visuals would be helpful

### PR DESCRIPTION
**Pull request by Vrindagupta6828**
Fixes #29

### CODE REVIEW

**Comment by Vrindagupta6828** [comment]
@ADI10HERO  Please review it.

File: example_json/postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+						""
+					]
+				},
+				"description": "DLMML "
+			},
+			"response": []
+		}
+	],
+	"protocolProfileBehavior": {}
+}
```
Comment (ADI10HERO) [inline example_json/postman_collection.json:44]:
```suggestion
}

```

**Review by ADI10HERO — APPROVED** [review]
Thank you so much.

**Comment by ADI10HERO** [comment]
@Vrindagupta6828  PLease make sure your "]" closing box bracket is followed by a white space  `" "` in the PR title

**Comment by ADI10HERO** [comment]
give @Vrindagupta6828 25 points

**Comment by ADI10HERO** [comment]
give @Vrindagupta6828 25 points

**Pull request merged**